resources/WELCOME.md

```diff
@@ -3,72 +3,70 @@
 
 Welcome to **Skyra Lounge**!
 _ _
-I'm an AIO Discord Bot focused on moderation, social, and utility that was born out of curiosity. My name comes from a joke a friend of my creator made by joining "Kyra" with "SkyNET", resulting in "SkyraNET". The joke kinda stuck around and became my name. An alternative name was "Shiny", which became the name of the currency instead.
+We are a developer community that produces both Discord bots (also called *ArchId Network Bots*) and libraries of various kinds that are released on [npm](https://www.npmjs.com) under the `@skyra/` prefix.
 _ _
-Want to know even more cool information about me? My creator Kyra has been writing my lore since he was only two and a half years old! Big parts are currently being written down for others to read, suggest and contribute to. If you want to learn some of it use `s!aifact`.
+__All of our source code is available under [`skyra-project`](https://github.com/skyra-project) on [GitHub](https://github.com).__ We also have a [website](https://skyra.pw)!
```

Comment:
when is the org being renamed to archid-network

Reply:
I don't know, the website is `skyra.pw`, the packages are `@skyra/*`, it would be fairly inconsistent to see the org's name being different.

I did consider it, but ArchId Network refers only to the bots, while Skyra Project refers to ArchId Network *and* the packages, all of which we support.

Reply:
Fair